src/python/zquantum/qaoa/farhi_ansatz_test.py

```diff
@@ -111,10 +111,11 @@ def test_generate_circuit(self):
     def test_get_symbols(self):
         # Given
         target_symbols = [
-            sympy.Symbol("beta_0"),
             sympy.Symbol("gamma_0"),
-            sympy.Symbol("beta_1"),
+            sympy.Symbol("beta_0"),
```

Comment:
If you change `get_symbols` as I suggested you should also revert these tests :)